query_rest.go

```diff
@@ -13,10 +13,10 @@ import (
 
 func (s *ChainService) queryRestPeers(query *cfiltersQuery) {
 	quit := make(chan struct{})
-	client := &http.Client{Timeout: QueryTimeout}
+	client := &http.Client{}
 	validPeers := make([]int, 0, len(s.restPeers))
 	for i, p := range s.restPeers {
-		if p.failures == 0 || time.Since(p.lastFailure) > 10*time.Second {
```

Comment:
Why change this?

Reply:
Good point, perhaps we do not need to change this.